File: Week1_EnginneringConcepts/Code/E-CommercePlatform/Product.java
class Product{
    int productId;
    String productName;
    String productCatergory;

public Product(int productId,String productName,String productCategory){
    this.productId=productId;
    this.productName=productName;
    this.productCatergory=productCategory;
}
public String result(){
    return "Product ID: " +productId+", Name: " +productName+", Category: "+productCatergory;
}
}

File: Week1_EnginneringConcepts/Code/2_ECommercePlatform/Product.cs
using System;

public class Product
{
    public int ProductId { get; set; }
    public string ProductName { get; set; }
    public string Category { get; set; }

    public Product(int id, string name, string category)
    {
        ProductId = id;
        ProductName = name;
        Category = category;
    }

    public override string ToString()
    {
        return $"ID: {ProductId}, Name: {ProductName}, Category: {Category}";
    }
}
